tests/resources/ankaios_library.py

```diff
@@ -343,9 +344,29 @@ def internal_check_all_control_interface_requests_succeeded(tmp_folder):
 def internal_check_all_control_interface_requests_failed(tmp_folder):
     output = read_yaml(path.join(tmp_folder, "output.yaml"))
     for test_number,test_result in enumerate(output):
-        test_result = test_result["result"]["value"]["type"] != "Ok"
+        try:
+            test_result = test_result["result"]["value"]["type"] != "Ok"
+        except:
+            test_result["result"] != "Ok"
```

Comment:
We should add a different function for the check that the Control Interface is not there. This should be done already at the place where the `No Rules` test is specified. 

The expectation there should be something like: `Then the controller workload has no access to Control Interface`. At the end the whole chain can lead to new function here in the python lib that actually assumes that `test_result["result"]["type"] != "NoAPI"`.

The changes here can be reverted with this approach, but I would suggest to make the check here more explicit and not assume `!= "Ok"`, but rather `== "Err"`

Reply:
Done.